image-micro-service/src/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const resize = require('./resize')
+const server = express()
+
+server.get('/', (req, res) => {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T21:28:36Z" title="Friday, December 20th 2019, 10:28:36 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const resize = require('./resize')
+const server = express()
+
+server.get('/', (req, res) => {
```


This looks great! Maybe we can merge this into the backend in order to quickly merge the responsive image feature?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-24T11:23:44Z" title="Friday, January 24th 2020, 12:23:44 pm +01:00">Jan 24, 2020</time>_

---


how would this work with what we have in the backend? 
I'm looking at this as a potential solution for https://github.com/Human-Connection/Human-Connection/issues/1922
but I don't think we would use `server.get('/')`
I couldn't easily figure out what it should be... I think maybe `server.get('/api/')`, oder? @roschaefer